Demo/Demo/ViewModels/VenmoPaymentsViewModel.swift

```diff
@@ -1,3 +1,31 @@
+import CorePayments
 import Foundation
 
-class VenmoPaymentsViewModel: ObservableObject { }
+class VenmoPaymentsViewModel: ObservableObject {
+    
+    let configManager = CoreConfigManager(domain: "Venmo Payments")
+    
+    @Published var state = VenmoState()
```

Comment:
Take it our leave it for this PR, but we started a transition to use a new `CurrentState` enum across all of the views vs individualized views with custom states. If you take a look at the PayPal Web Payments views you can see where that was in motion. Not a blocker, would just be nice to eventually share duplicated states with a shared interface. Can totally be a follow up or something we tackle when we address the other view since only PayPal Web was updated thus far.

Reply:
Updated, thanks for your feedback: [33d3d46](https://github.com/paypal/paypal-ios/pull/286/commits/33d3d46d5edc9ecaa532cac49329d3b6008ff3dd)